requirements.txt

```diff
@@ -1 +1,2 @@
-pygerduty==0.25
+pypd==0.1.0
+urllib3[secure]
```

Comment:
I'd planned to take this one out.. it was the recommended solution to fix the insecure https warning. But doesn't seem to work when installed as a pack. Newer pythons don't have this issue.

```
|                 |     "stderr": "/opt/stackstorm/st2/lib/python2.7/site-       |
|                 | packages/requests/packages/urllib3/util/ssl_.py:318:         |
|                 | SNIMissingWarning: An HTTPS request has been made, but the   |
|                 | SNI (Subject Name Indication) extension to TLS is not        |
|                 | available on this platform. This may cause the server to     |
|                 | present an incorrect TLS certificate, which can cause        |
|                 | validation failures. You can upgrade to a newer version of   |
|                 | Python to solve this. For more information, see https://urll |
|                 | ib3.readthedocs.io/en/latest/security.html#snimissingwarning |
|                 | .                                                            |
|                 |   SNIMissingWarning                                          |
|                 | /opt/stackstorm/st2/lib/python2.7/site-                      |
|                 | packages/requests/packages/urllib3/util/ssl_.py:122:         |
|                 | InsecurePlatformWarning: A true SSLContext object is not     |
|                 | available. This prevents urllib3 from configuring SSL        |
|                 | appropriately and may cause certain SSL connections to fail. |
|                 | You can upgrade to a newer version of Python to solve this.  |
|                 | For more information, see https://urllib3.readthedocs.io/en/ |
|                 | latest/security.html#insecureplatformwarning.                |
|                 |   InsecurePlatformWarning                                    |
```